protos/protos/public/modeldb/metadata/MetadataService.proto

```diff
@@ -50,6 +58,35 @@ message DeleteLabelsRequest {
     }
 }
 
+message GetMetadataRequest {
```

Comment:
Can we give this a more meaningful name? "Metadata" is way too broad.

Reply:
already changes to Properties in the actual branch supporting this